### PR TITLE
[VTAdmin] Change vars from REACT_APP to VITE

### PR DESCRIPTION
**Pull request by notfelineit**
With https://github.com/vitessio/vitess/pull/12831, we're migrating VTAdmin from create-react-app to vite. One of the changes is that all `REACT_APP_*` vars are changed to `VITE_*`

### CODE REVIEW

**Review by frouioui — APPROVED** [review]

**Comment by frouioui** [comment]
Went through the repository and it does not seem like we need to change anything, i'll let @GuptaManan100 confirm though.

**Review by GuptaManan100 — APPROVED** [review]
Changes look good to me!

File: pkg/controller/vitesscluster/reconcile_vtadmin.go [hunk 1]
```diff
@@ -341,12 +341,12 @@ func (r *ReconcileVitessCluster) createWebConfigSecret(ctx context.Context, vt *
 	// Variables to hold the key, value and secret name to use
 	configKey := vtadmin.WebConfigFileName
 	configVal := fmt.Sprintf(`window.env = {
-     'REACT_APP_VTADMIN_API_ADDRESS': "%s",
-     'REACT_APP_FETCH_CREDENTIALS': "omit",
-     'REACT_APP_ENABLE_EXPERIMENTAL_TABLET_DEBUG_VARS': false,
-     'REACT_APP_BUGSNAG_API_KEY': "",
-     'REACT_APP_DOCUMENT_TITLE': "",
-     'REACT_APP_READONLY_MODE': %s,
```
Comment (GuptaManan100) [inline pkg/controller/vitesscluster/reconcile_vtadmin.go:349]: We should probably keep the old env variables for one release to ensure backward compatibility.
We can drop these after 2.10.0

**Comment by frouioui** [comment]
Merged `main` (since #398 is merged) onto this Pull Request to unblock CI.

File: pkg/controller/vitesscluster/reconcile_vtadmin.go [hunk 2]
```diff
@@ -341,12 +341,12 @@ func (r *ReconcileVitessCluster) createWebConfigSecret(ctx context.Context, vt *
...
+     'VITE_VTADMIN_API_ADDRESS': "%s",
+     'VITE_FETCH_CREDENTIALS': "omit",
+     'VITE_ENABLE_EXPERIMENTAL_TABLET_DEBUG_VARS': false,
+     'VITE_BUGSNAG_API_KEY': "",
+     'VITE_DOCUMENT_TITLE': "",
+     'VITE_READONLY_MODE': %s,
```
Comment (notfelineit) [inline pkg/controller/vitesscluster/reconcile_vtadmin.go:349]:
```suggestion
     'VITE_READONLY_MODE': %s,
     'REACT_APP_VTADMIN_API_ADDRESS': "%s",
     'REACT_APP_FETCH_CREDENTIALS': "omit",
     'REACT_APP_ENABLE_EXPERIMENTAL_TABLET_DEBUG_VARS': false,
     'REACT_APP_BUGSNAG_API_KEY': "",
     'REACT_APP_DOCUMENT_TITLE': "",
     'REACT_APP_READONLY_MODE': %s,
```

**Pull request merged**